server/package.json

```diff
@@ -13,7 +13,9 @@
     "test": "jest",
     "ci": "jest --coverage",
     "generate-secret": "./scripts/generate-secret.js",
-    "dev:e2e": "NODE_ENV=e2e PORT=8765 nodemon src/app.js"
+    "dev:e2e": "NODE_ENV=e2e PORT=8765 nodemon src/app.js",
+    "ymrm": "yarn migrate:latest && yarn migrate:rollback && yarn migrate:latest",
```

Comment:
I would actually suggest setting up `ymrm` as an alias in ZSH instead of as a script in your project. 

You can do this by opening your zshrc file:
`code ~/.zshrc`
then adding the below line
`alias ymrm="yarn migrate:latest && yarn migrate:rollback && yarn migrate:latest"`
then `source ~/.zshrc` or restart your terminal
now you have access to this shortcut command in ALL projects!